src/Actions/Actions/LoadAction.cs

```diff
@@ -53,9 +57,22 @@ internal static LoadActionParts LoadSnapshotZip(string path)
                 SnapshotMetaInfo meta = SnapshotMetaInfo.DeserializeFromStream(metadataPart);
                 metadataPart.Close();
 
+                IReadOnlyDictionary<int, CustomProperty> CustomProperties;
+                try
+                {
+                    var customPropertiesPart = (from p in parts where p.Uri.OriginalString == "/" + SaveAction.customPropsFileName select p.GetStream()).First();
+                    CustomProperties = LoadCustomProperties(customPropertiesPart);
+                    customPropertiesPart.Close();
+                }
+                catch (InvalidOperationException e)  // Gets thrown if custom properties map doesn't exist in file
+                {
+                    e.ReportException();
```

Comment:
We can probably skip reporting this exception. It's an expected scenario for older files